src/main/java/org/dsa/iot/knx/Utils.java

```diff
@@ -0,0 +1,10 @@
+package org.dsa.iot.knx;
+
+import java.util.Arrays;
+
+public class Utils {
+	public static <E> String[] enumNames(Class<E> enumData) {
+		String valuesStr = Arrays.toString(enumData.getEnumConstants());
```

Comment:
Can't you already do that with Enums? something like `Enum.values` or `Enum.names` ?